cf-agent/verify_files_utils.c

```diff
@@ -1281,8 +1281,8 @@ bool CopyRegularFile(EvalContext *ctx, const char *source, const char *dest, con
     char backup[CF_BUFSIZE];
     char new[CF_BUFSIZE], *linkable;
     int remote = false, backupisdir = false, backupok = false, discardbackup;
-    struct stat s;
-    struct stat dstat;
+    struct stat dest_stat;
+    struct stat new_stat;
```

Comment:
These declarations should be moved close to the place these variables are used for the first time.

Reply:
Agree. :+1: